### PR TITLE
test: add test in Makefile

### PR DESCRIPTION
**Pull request by sunyuan3**
Signed-off-by: Yuan Sun <yile.sy@alibaba-inc.com>

**1.Describe what this PR did**
add test in Makefile

**2.Does this pull request fix one issue?** 
No

**3.Describe how you did it**
add test and integration-test in Makefile

**4.Describe how to verify it**
make integration-test
make test

**5.Special notes for reviews**
None

### CODE REVIEW

**Comment by allencloud** [comment]
Please add a prefix in the PR title, such as: test, doc, bugfix, and so on. @sunyuan3

File: .travis.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@ before_install:
 script:
   # PATH is not set under sudo, then we set up path mannually.
   # Ref https://unix.stackexchange.com/questions/83191/how-to-make-sudo-preserve-path
```
Comment (0x04C2) [inline .travis.yml]: Could we move the comment to the Makefile?

Comment (sunyuan3) [reply .travis.yml]: Yes, I have update it.

**Comment by sunyuan3** [comment]
ping @allencloud @Letty5411 
The known issue leads to the failure of CI. 
Please review this PR.

**Comment by Letty5411** [comment]
@sunyuan3 LGTM

**Comment by allencloud** [comment]
Please rebase to the latest master, since PR https://github.com/alibaba/pouch/pull/436 which solved flaky test has been merged. Thanks.

**Comment by sunyuan3** [comment]
ping @allencloud I have rebased this pr.

**Pull request merged**